secp256k1.nim

```diff
@@ -269,7 +284,10 @@ proc secp256k1_ecdsa_recoverable_signature_parse_compact*(
 
 proc secp256k1_ecdh*(ctx: ptr secp256k1_context; output32: ptr cuchar;
                      pubkey: ptr secp256k1_pubkey;
-                     input32: ptr cuchar): cint {.secp.}
+                     privkey: ptr cuchar,
```

Comment:
It would be nice if upstream was consistent in the use of `seckey` and `privkey` but it might be an ECDH only thing, this sure is confusing: https://github.com/bitcoin-core/secp256k1/blob/1e6f1f5ad5e7f1e3ef79313ec02023902bf8175c/contrib/lax_der_privatekey_parsing.h#L37-L63